recipes/xqilla/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.40.0"
+
+
+class ConanXqilla(ConanFile):
+    name = "xqilla"
+    
+    description = (
+        "XQilla is an XQuery and XPath 2 library and command line utility written in C++, implemented on top of the Xerces-C∞ library"
```

Comment:
```suggestion
        "XQilla is an XQuery and XPath 2 library and command line utility written in C++, implemented on top of the Xerces-C library"
```

Not sure if that char can create problems depending the machine.